src/i_sndfile.c

```diff
@@ -519,10 +552,21 @@ boolean I_SND_LoadFile(void *data, ALenum *format, byte **wavdata,
         return false;
     }
 
-    *wavdata = local_wavdata;
     *size = num_frames * file.frame_size / file.sfinfo.channels;
     *freq = file.sfinfo.samplerate;
 
+    // Fade in sounds that start at a non-zero amplitude to prevent clicking.
+    if (*format == AL_FORMAT_MONO16)
```

Comment:
What if we load 8-bit WAV/VOC sound samples with libsndfile?

Reply:
Sounds like a good idea. I think there could even be some simplifying of `I_OAL_CacheSound()` in i_oalsound.c and it makes sense to allow 8-bit WAV in wads without having to convert to Doom format, right? I will take a closer look some time this weekend.

Reply:
We do not support loading 8-bit WAV files with libsndfile. I hadn't thought of that. I think it is beyond the scope of this PR.

Reply:
I think the 8-bit non-Doom WAV is loaded anyway, just converted to 16-bit with libsndfile.

Reply:
Yes, I think non-Doom-format 8-bit WAV files still go through libsndfile.

Reply:
Yep, seems to work fine.